CMakeLists.txt

```diff
@@ -337,6 +337,9 @@ cmake_dependent_option(USE_CCACHE "Attempt using CCache to wrap the compilation"
 option(WERROR "Build with -Werror supported by the compiler" OFF)
 option(USE_COREML_DELEGATE "Use the CoreML backend through delegate APIs" OFF)
 option(USE_PER_OPERATOR_HEADERS "Whether ATen should generate separate headers for each operator" ON)
+cmake_dependent_option(
```

Comment:
@malfet Any chance you can spot what I'm doing wrong here? If you look at failures on this PR's CI, I am observing from cmake.summary that 
INTERN_BUILD_MOBILE: ON
BUILD_LAZY_TS_BACKEND: ON
and I can't figure out why my flag isn't getting turned off under these builds.